dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -293,3 +294,15 @@ CWPKI2063E.useraction=Review the provided terms of service.
 CWPKI2064I=CWPKI2064I: The ACME service retrieved the certificate with the {0} serial number from the {1} URI in {2} seconds.
 CWPKI2064I.explanation=The ACME service successfully requested a certificate.
 CWPKI2064I.useraction=No action is required.
+
+CWPKI2065W=CWPKI2065W: The ACME Service failed to automatically renew the certificate with the {0} serial number. The request is scheduled to try again in {3}. The certificate expires on {1}. The renew request error is ''{2}''.
+CWPKI2065W.explanation=The ACME Service tried to renew a certificate but encountered an error. The ACME Service continues to request a new certificate until a new certificate is issued.
+CWPKI2065W.useraction=Review the error message for details on the failure.
+
+CWPKI2066E=CWPKI2066E: The ACME Service failed to automatically renew the certificate with the {0} serial number. The certificate is revoked. The request is scheduled to try again in {2}. The renew request error is ''{1}''.
+CWPKI2066E.explanation=The ACME Service tried to renew a certificate but encountered an error. The ACME Service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.
+CWPKI2066E.useraction=Review the error message for details on the failure. 
+
+CWPKI2067I=CWPKI2067I: The certificate with the {0} serial number is revoked. The ACME service requests a new certificate from the ACME certificate authority at the {1} URI.
+CWPKI2067I.explanation=The ACME Service detected that the certificate is revoked and automatically requests a new certificate.
```

Comment:
`The ACME Service detected that the certificate is revoked and automatically requests a new certificate`.
 
need to get the verb tenses to agree- I think it wil read better like this:

`When the ACME Service detects that the certificate is revoked, it automatically requests a new certificate.`

Reply:
Done